cmd/spruce/main_test.go

```diff
@@ -250,6 +250,53 @@ map:
 			So(stderr, ShouldEqual, "")
 		})
 
+		Convey("Should not evaluate spruce logic appart from (( prune )) when --no-eval", func() {
```

Comment:
Can you change the name of the test case?  It seems to imply that `(( prune ... ))` _should_ be processed.